etc/column_mappings.yaml

```diff
@@ -296,16 +300,16 @@ samples:
     - 2D_barcode
     - current_lims_status
     - library_type
-    - species
-    - { column_def: 'clean_yield_in_gb', title: 'Yield (Gb)', fmt: { type: 'float', min: {field: 'Required Yield (Gb)', default: 120} } }
-    - { column_def: 'clean_yield_q30_in_gb', title: 'Yield Q30 (Gb)', fmt: { type: 'float', min: {field: 'Yield for Quoted Coverage (Gb)', default: 95} } }
```

Comment:
I felt this was misleading, and have also been bitten by this in the past when reviewing data